megamek/src/megamek/client/ui/swing/MovementDisplay.java

```diff
@@ -878,9 +879,13 @@ private void updateButtons() {
             } else {
                 getBtn(MoveCommand.MOVE_FORTIFY).setEnabled(true);
             }
+        } else if (isTank && (ce.hasWorkingMisc(MiscType.F_BULLDOZER)
```

Comment:
Suggest using EquipmentTypeLookup.BULLDOZER and EquipmentTypeLookup.BACKHOE here (I have no idea which equipment would be "ruled equivalent" as per TO, so I'd rather be specific)

(A real good use of those flags would be adding a TRENCH_CAPABLE flag to Tools, Vibroshovel, Bulldozer and Backhoe and checking that flag, but that's not something for a lil change request)

Reply:
In that case this whole if {} block could be reduced to one line

Reply:
Yeah, that would make it easier. F_TOOLS is a flag itself, so instead we would just need to add F_TRENCH_CAPABLE to those three equipment for now. It would also eliminate the need for testing the sub types.